lib/utils/index.js

```diff
@@ -512,6 +512,9 @@ class Utils {
     });
   }
 
+  // eslint-disable-next-line
+  static scriptFn(e,a,v){try {if(e&&typeof e.setAttribute=='function'){e.setAttribute(a,v)} return true} catch(err){return {error:err.message,message:err.name+': '+err.message}}};
+
```

Comment:
Can we rename this?

Reply:
yes I think we should, how about `setAttributeFn`